sdk/evaluation/azure-ai-evaluation/setup.py

```diff
@@ -73,6 +73,9 @@
         "nltk>=3.9.1",
         "azure-storage-blob>=12.10.0",
     ],
+    extras_require={
+        "redteam": ["pyrit==0.7.0"]
```

Comment:
we'll need to update that once we have a new release

Reply:
@romanlutz will update once you have new release